src/System.IO.Pipelines/src/System/IO/Pipelines/PipeWriter.cs

```diff
@@ -48,10 +48,20 @@ public abstract partial class PipeWriter : IBufferWriter<byte>
         /// <summary>
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeWriter"/>.
         /// </summary>
+        /// <param name="leaveOpen">Optional flag indicating disposing returned <see cref="Stream"/> won't complete <see cref="PipeWriter"/>.</param>
```

Comment:
```suggestion
        /// <param name="leaveOpen"><see langword="true" /> to leave the pipe open after the <see cref="System.IO.Stream" /> object is disposed; otherwise, <see langword="false" />.</param>
```